### DISCUSSION TITLE
Verbose logging in Home Assistant Add-On

### DISCUSSION THREAD

**Discussion by svh1985** [mKeRix/room-assistant#636] [Q&A]
Hello,

How can I enable verbose logging in the Home Assistant Add-On?

Thanks!

**Comment by mKeRix** [comment]
Unfortunately this isn't possible atm, unless you edit the add-on container command with the Portainer add-on (or something similar). Since we had to change the config storage of the add-on anyway I think it'll be easier to implement this now though - so feel free to open this as an issue for better visibility/tracking.
